### PR TITLE
Show attendences

### PR DESCRIPTION
**Pull request by okwast**

### CODE REVIEW

File: backend/metadata/databases/default/tables/public_Person.yaml [hunk 1]
```diff
@@ -55,6 +55,7 @@ select_permissions:
   role: anonymous
 - permission:
     columns:
+    - AuthId
```
Comment (okwast) [inline backend/metadata/databases/default/tables/public_Person.yaml:58]: Is there a special reason for this? I personally find it difficult to expose the secret keys of every user to the public.

File: frontend/components/course/CoursePageStudentView.tsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ export const CoursePageStudentView: FC<IProps> = ({ course }) => {
           className="my-24"
           leftTop={
             <div className="flex flex-1">
-              <Attendences course={course} />
+              <Attendences course={course as any} />
```
Comment (okwast) [inline frontend/components/course/CoursePageStudentView.tsx]: Any - You shall not pass!

File: frontend/queries/courseWithEnrollment.ts [hunk 3]
```diff
@@ -6,12 +6,25 @@ import { ENROLLMENT_FRAGMENT } from "./enrollmentFragment";
...
+        CourseId
+        Description
+        Location
+        Start
+        Title
+        Attendences(where: { Person: { AuthId: { _eq: $authId } } }) {
+          Id
+          Attending
+        }
+      }
```
Comment (okwast) [inline frontend/queries/courseWithEnrollment.ts:23]: Awesome this is working 🎉 
I think I would still prefer to have the session thing in a fragment as they are a copy here right now.

File: backend/metadata/databases/default/tables/public_Person.yaml (hunk 1 above)
Comment (fschindler) [reply backend/metadata/databases/default/tables/public_Person.yaml:58]: We need this `AuthId` the filter of Attendances: `Attendences(where: { Person: { AuthId: { _eq: $authId } } })`.
IMHO the `AuthId` isn't a secret.

Asked in passing: Is there a typo in the database `Attendences`?

File: frontend/components/course/CoursePageStudentView.tsx (hunk 2 above)
Comment (fschindler) [reply frontend/components/course/CoursePageStudentView.tsx]: This was still WIP.

File: frontend/queries/courseWithEnrollment.ts (hunk 3 above)
Comment (fschindler) [reply frontend/queries/courseWithEnrollment.ts:23]: Yes, after finalising WIP there now is a fragment.

File: backend/metadata/databases/default/tables/public_Person.yaml (hunk 1 above)
Comment (okwast) [reply backend/metadata/databases/default/tables/public_Person.yaml:58]: Yep, seems it should be `Attendances`.

Comment (steffen74) [reply backend/metadata/databases/default/tables/public_Person.yaml:58]: The correction is already part of the Backend Changes in issue #49

**Comment by okwast** [comment]
Great job! Code looks really good and working perfectly fine!
Ready to 🚀 

PS: I should not have opened this. Now I can't approve this. @fschindler please do so and feel free to merge this :)

**Comment by fschindler** [comment]
@okwast thank's for your feedback, I'll approve and merge this one.

**Review by fschindler — APPROVED** [review]

**Pull request merged**